react/features/video-menu/components/web/LocalVideoMenuTriggerButton.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+
+import React from 'react';
+
+import { Icon, IconMenuThumb } from '../../../base/icons';
+import { Popover } from '../../../base/popover';
+import { connect } from '../../../base/redux';
+import { getLocalVideoTrack } from '../../../base/tracks';
+import { getCurrentLayout, LAYOUTS } from '../../../video-layout';
+
+import FlipLocalVideoButton from './FlipLocalVideoButton';
+import VideoMenu from './VideoMenu';
+
+/**
+ * The type of the React {@code Component} props of
+ * {@link LocalVideoMenuTriggerButton}.
+ */
+type Props = {
+
+    /**
+     * The position relative to the trigger the local video menu should display
+     * from. Valid values are those supported by AtlasKit
+     * {@code InlineDialog}.
+     */
+    _menuPosition: string,
+
+    /**
+     * Whether to display the Popover as a drawer.
+     */
+    _overflowDrawer: boolean,
+
+    /**
+     * Shows/hides the local video flip button.
+     */
+    _showLocalVideoFlipButton: boolean
+};
+
+/**
+ * React Component for displaying an icon associated with opening the
+ * the video menu for the local participant.
+ *
+ * @param {Props} props - The props passed to the component.
+ * @returns {ReactElement}
+ */
+function LocalVideoMenuTriggerButton(props: Props) {
+    return (
+        props._showLocalVideoFlipButton
```

Comment:
If you invert the check you could return null early and save you an indentation level for some extra readability.

Reply:
Hmm... I'm not sure I understand.

Do you mean instead of using the conditional operator to use `!props._showLocalVideoFlipButton && <Popover ... />`?

Reply:
You can do this:

```
if (!props._showLocalVideoFlipButton) {
    return null;
}

return (
    <Popover...
);
```